spec/webauthn/public_key_credential_spec.rb

```diff
@@ -92,5 +92,68 @@
         }.to raise_error(WebAuthn::ChallengeVerificationError)
       end
     end
+
+    context "when clientExtensionResults is received" do
+      let(:public_key_credential) do
+        WebAuthn::PublicKeyCredentialWithAttestation.new(
+          type: type,
+          id: id,
+          raw_id: raw_id,
+          client_extension_outputs: { "appid" => "true" },
+          response: attestation_response
+        )
+      end
+
+      it "works" do
+        expect(public_key_credential.verify(challenge)).to be_truthy
+
+        expect(public_key_credential.client_extension_outputs).to eq({ "appid" => "true" })
+      end
+    end
+
+    context "when authentication extension input" do
+      context "is not received" do
+        let(:attestation_response) do
+          allow_any_instance_of(WebAuthn::FakeAuthenticator::AuthenticatorData)
+            .to receive(:extensions)
+            .and_return(nil)
```

Comment:
Even though I'm not a big fan of using stubs for testing, I couldn't find another way of manually set up the `attestation_response` with the specific extensions that I wanted. 
Obviously, any suggestion is appreciated!

Reply:
Yeah I gave a little thinking about that, but couldn't came up with an easy implementation 😕 
Anyway, I will go ahead and see If can find a solution 🙂

Reply:
Have we made any attempt to replace this stub?

Reply:
Not yet, I will try as soon as I can 🙂